### ISSUE TITLE
Android notifications will not disable

### ISSUE THREAD

**Issue by aaronkaz** [Vydia/react-native-background-upload#214]
This option:

```
notification: {
  enabled: false,
},
```

seems to make no difference whether I toggle `true` or `false`.  I see Silent Notifications (API 29) for all uploads regardless of the value.  Any ideas how to turn these off?

**Comment by dolphinflow86** [comment]
Same here. @aaronkaz did you solve this issue?

**Comment by aaronkaz** [comment]
@dolphinflow86 No, I never did.  This wasn't the only big issue I ran into, so I've just abandoned using this package altogether.

**Comment by reime005** [comment]
On Android versions >= 8.0 Oreo it is required by Google's policy to display a notification when a background service (such as uploading a file in the background) runs

**Comment by olegzecamp** [comment]
Same, Notifications show regardless of the notification.enabled parameter value.

**Comment by bneigher** [comment]
@reime005 isn't that only the case if you're using the Android File Manager?  It should still be possible to do this with `DOWNLOAD_WITHOUT_NOTIFICATION` permission.  Otherwise we would see even native apps going nuts when we upload files via apps

**Comment by bneigher** [comment]
Does anyone know how to contribute to the android native code here?  I for the life of me can't figure out where to look to fix this

**Comment by reime005** [comment]
@bneigher I don't think it is fixable...but the android code is in the /android folder if you mean that

**Comment by mar-bi** [comment]
I disabled it (it's kind of "disabled" - notification is removed right after a successful upload. The user almost doesn't notice it) with this combination of parameters:
```
notification: {
        enabled: true,
        autoClear: true,
      },
```

**Comment by AlexCernik** [comment]
> I don't think it is fixable...but the android code is in the /android folder if you mean that

Notifications not autoClear, is persisted
